build.gradle

```diff
@@ -89,6 +90,18 @@ jacocoTestReport {
   }
 }
 
+project.tasks['sonarqube'].dependsOn test, integration
+
+sonarqube {
+  properties {
+    property "sonar.projectName", "Reform :: Bulk Scan Orchestrator"
+    property "sonar.jacoco.reportPath", "${project.buildDir}/jacoco/test.exec"
+
+    // TODO: uncomment the following line when integration tests are created
+    // property "sonar.jacoco.itReportPath", "${project.buildDir}/jacoco/integration.exec"
```

Comment:
it may be safer to add an empty integration package (or with dummy test) ;)

Reply:
I've started considering that, too. Will do.